### ISSUE TITLE
Ability to delete Trauma or Pathology

### ISSUE THREAD

**Issue by clegarde** [spawaskar-cora/cora-docs#55]
Need the ability to delete a trauma or pathology, particularly during the inventory stage. Either delete on the general trauma page (as noted in the picture) or after clicking on the specific trauma.
![traumapath_delete](https://user-images.githubusercontent.com/7350474/36266577-63a8895a-1237-11e8-9991-aadca82c2f57.PNG)

**Comment by SachinPawaskarUNO** [comment]
This issue has been addressed via SachinPawaskarUNO/cora#430 & SachinPawaskarUNO/cora#431.
@clegarde Please verify and close out if good to go.

**Comment by clegarde** [comment]
@SachinPawaskarUNO, Tested - I can delete a trauma or pathology that I enter. I cannot delete a trauma or pathology on a bone that someone else entered. Even if I was the one that added the trauma or pathology. The element I tested it on was marked "Inventory Completed," so I think that's why. I think this is ok though. I got the following message after I clicked delete:
![deletetrauma](https://user-images.githubusercontent.com/7350474/36313172-626a531a-12f6-11e8-84f5-e55dfbc06ef2.PNG)
Also, on Internet Explorer, I cannot view the delete icon.
